### PR TITLE
[9.0] [BACKPORT] Purchase Packaging

### PR DESCRIPTION
**Pull request by ThomasBinsfeld**
Backport of https://github.com/OCA/stock-logistics-warehouse/tree/10.0/purchase_packaging

It includes two approved v10 PR's on the module:
 * https://github.com/OCA/stock-logistics-warehouse/pull/453
 * https://github.com/OCA/stock-logistics-warehouse/pull/547

I does not include two other PR's since they doesn't seem ready:
 * https://github.com/OCA/stock-logistics-warehouse/pull/349
 * https://github.com/OCA/stock-logistics-warehouse/pull/493

----------
This PR depends on https://github.com/OCA/stock-logistics-warehouse/pull/576

### CODE REVIEW

**Review by gurneyalex — APPROVED** [review]

**Review by lmignon — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by rafaelbn — APPROVED** [review]
:+1: thanks

**Comment by rousseldenis** [comment]
@ThomasBinsfeld #576 merged

**Comment by ThomasBinsfeld** [comment]
This PR is ready to be merged.

**Comment by rousseldenis** [comment]
@ThomasBinsfeld Wait for travis and merging

File: .travis.yml [hunk 1]
```diff
@@ -29,6 +29,9 @@ virtualenv:
   system_site_packages: true
 
 install:
+  #  FIXME: remove this after merge
```
Comment (rousseldenis) [inline .travis.yml]: @ThomasBinsfeld Could you remove this as merged ?

Comment (ThomasBinsfeld) [reply .travis.yml]: Yes, done.

**Pull request merged**